auth-web/src/components/pay/LinkedShortNameTable.vue

```diff
@@ -41,6 +41,18 @@
           </v-icon>
         </v-btn>
       </template>
+      <template #item-slot-accountName="{ item }">
+        <span>{{ item.accountName }}</span>
+        <v-chip
+          v-if="item.cfsAccountStatus && item.cfsAccountStatus !== 'ACTIVE'"
```

Comment:
shouldn't we look for `FROZEN`, it's more decisive?

Reply:
yeah this wont work for pending pad activation...
![image](https://github.com/bcgov/sbc-auth/assets/3484109/2f1a5230-e900-4c27-bb79-2d9c74ca7787)

Reply:
INACTIVE is rare ish.. maybe people switching from PAD -> BCOL.. would have to look it up

Reply:
`select * from cfs_accounts where account_id in (select id from payment_accounts where payment_method = 'DRAWDOWN') order by account_id;`

Looks like they are INACTIVE?

Reply:
fixed